frontend/styles/views/main/content.css

```diff
@@ -1,9 +1,123 @@
+/**
+ * Stylesheet for myProfileView
+ *
+ *@added Monika Martius und Vanessa Skowronksy
+ *@Version 1.0
+ *@since 07.01.2021
+ */
+
+#content-view_blue {
+    width: 80%;
+    height: 315%;
+    background: linear-gradient(180deg, rgba(0, 133, 255, 0.44) 0%, rgba(12, 127, 233, 0.355) 43.12%, rgba(14, 89, 203, 0) 100%);
+    color: #0D5699;
+    font-weight: bold;
+    font-family: "Gothic A1", serif;
+    font-style: normal;
+    font-size: 24px;
+    line-height: 30px;
+    position: absolute;
+    left: 4%;
+    /*border: 1px solid #9E9E9E;*/
+}
+
+#ourCompanyView{
+    width: 80%;
+    height: 315%;
+    background: linear-gradient(180deg, rgba(216, 131, 131, 0.5) 0%, rgba(196, 196, 196, 0) 105.33%);
+    color: #A00505;
+    font-weight: bold;
+    font-family: "Gothic A1", serif;
+    font-style: normal;
+    font-size: 24px;
+    line-height: 30px;
+    position: absolute;
+    left: 4%;
+}
 
+#projektView{
+    width: 80%;
+    height: 315%;
+    background: linear-gradient(180deg, rgba(128, 70, 174, 0.5) 0%, rgba(196, 196, 196, 0) 100%);
+    color: #581092;
+    font-weight: bold;
+    font-family: "Gothic A1", serif;
+    font-style: normal;
+    font-size: 24px;
+    line-height: 30px;
+    position: absolute;
+    left: 4%;
+}
+
+#librarView{
+    width: 80%;
+    height: 315%;
+    background: linear-gradient(180deg, rgba(47, 124, 120, 0.5) 0%, rgba(196, 196, 196, 0) 100%);
+    color: #255b57;
+    font-weight: bold;
+    font-family: "Gothic A1", serif;
+    font-style: normal;
+    font-size: 24px;
+    line-height: 30px;
+    position: absolute;
+    left: 4%;
+}
 
-#myProfile-view {
+#serviceView{
+    width: 80%;
+    height: 315%;
+    background: linear-gradient(180deg, rgba(255, 96, 33, 0.5) 0%, rgba(196, 196, 196, 0) 100%);
+    color: #9b3a23;
+    font-weight: bold;
+    font-family: "Gothic A1", serif;
+    font-style: normal;
+    font-size: 24px;
+    line-height: 30px;
+    position: absolute;
+    left: 4%;
+}
+
+#communityView{
+    width: 80%;
+    height: 315%;
+    background: linear-gradient(180deg, rgba(240, 191, 44, 0.5) 0%, rgba(196, 196, 196, 0) 100%);
+    color: #7c6d30;
+    font-weight: bold;
+    font-family: "Gothic A1", serif;
+    font-style: normal;
+    font-size: 24px;
+    line-height: 30px;
+    position: absolute;
+    left: 4%;
+}
+
+.layout_data{
     width: 100%;
     height: 100%;
-    background: linear-gradient(180deg, rgba(131, 140, 216, 0.5) 0%, rgba(196, 196, 196, 0) 81.8%);
+    text-align: center;
+    justify-content: center;
+    flex-direction: row;
+    display: flex;
+    flex-wrap: wrap;
+    /*border: 1px solid #9E9E9E;*/
+}
+
+.contentView{
+    width: 80%;
+    position: absolute;
+    left: 1%;
+}
+
+.profileViewRight{
+    width: 340px !important;
+    height: 478px;
+    left: 50%;
+}
+
+.profileViewLeft{
+    width:500px !important;
+    height: 478px;
+    left: 7%;
 }
 
```

Comment:
Bitte die .pageContentPositon-Klasse aus mainView.css in content.css kopieren und alle Attribute, die auf allen Seiten gesetzt werden, dort zusammen fassen. Das spart viele Zeilen Code ein :-)